common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -75,6 +76,11 @@ def student_view(self, context):
 
         return fragment
 
+    def get_unit_display_name(self):  # pylint: disable=missing-docstring
```

Comment:
In fact, I'm not sure what this method is for. I'd remove it and replace callers with ``self.display_name_with_default``.